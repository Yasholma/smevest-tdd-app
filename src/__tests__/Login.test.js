import React from "react";
import { shallow } from "enzyme";
import Login from "../components/login/login.component";


describe("<Login />", () => {
    test("should render login component", () => {
        expect(shallow(<Login />)).toMatchSnapshot();
    })
})