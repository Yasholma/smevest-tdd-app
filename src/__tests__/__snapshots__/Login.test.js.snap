// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> should render login component 1`] = `
<div
  className="Login"
>
  <div
    className="Container"
  >
    <h2>
      Login To SmeVest
    </h2>
    <form
      className="Form"
      onSubmit={[Function]}
    >
      <div
        className="FormGroup"
      >
        <input
          id="email"
          name="email"
          onChange={[Function]}
          placeholder="Email Address"
          type="email"
          value=""
        />
      </div>
      <div
        className="FormGroup"
      >
        <input
          id="password"
          name="password"
          onChange={[Function]}
          placeholder="Password"
          type="password"
          value=""
        />
      </div>
      <div
        className="FormButtons"
      >
        <div
          className="OAuthLinks"
        >
          <h4>
            Login Options
          </h4>
          <FacebookIcon
            style={
              Object {
                "background": "#3b5998",
                "borderRadius": "50%",
                "color": "#fff",
                "cursor": "pointer",
                "margin": "0 5px",
                "padding": 5,
              }
            }
          />
          <MailIcon
            style={
              Object {
                "background": "#D44638",
                "borderRadius": "50%",
                "color": "#fff",
                "cursor": "pointer",
                "margin": "0 5px",
                "padding": 5,
              }
            }
          />
        </div>
        <button
          type="submit"
        >
          Login
        </button>
      </div>
      <p>
        Don't have an account yet?
         
        <Link
          to="/register"
        >
          Register
        </Link>
      </p>
    </form>
  </div>
</div>
`;
